scripts/zkai_sota_artifact_watchlist_gate.py

```
@@ -0,0 +1,954 @@
+        "systems_commitment",
+        "summary",
+        "paper_actions",
+        "validation_commands",
+        "non_claims",
+        "mutation_inventory",
+        "cases",
+        "case_count",
+        "all_mutations_rejected",
+    }
+    if set(payload) != expected_fields:
+        raise SotaWatchlistError("payload field set mismatch")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)